javascript/js-structure_quiz-app/components/Card/Card.js

```diff
@@ -0,0 +1,58 @@
+// import function bookmark
+import Bookmark from "../Bookmark/Bookmark.js";
+// import variable bookmark
+import cards from "../CardsData/CardsData.js";
```

Comment:
I think that cards could be { cards }. Because it is not a default function. Does it work in your console?

Reply:
Yes, because it is actually a default :) I moved it to its own file CardsData.js and made it as default there.